Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -139,6 +142,8 @@ public enum FeatureFlag: String, CaseIterable {
             false
         case .transcripts:
             false
+        case .gravatarChangeAvatar:
+            true
```

Comment:
I made it true by default, let me know if you think it needs to be false.

Reply:
Yes, that's right. This value is just used to set the initial value so that it's enabled or disabled prior to connecting to Firebase. We also tend to leave this value `false` while testing until we are mostly sure we want to release in whichever version this is going into.